src/core/prompts/responses.ts

```diff
@@ -205,6 +205,9 @@ Otherwise, if you have not completed the task and do not need additional informa
 
 	clineRulesFileInstructions: (cwd: string, content: string) =>
 		`# .clinerules\n\nThe following is provided by a root-level .clinerules file where the user has specified instructions for this working directory (${cwd.toPosix()})\n\n${content}`,
+
+	newTaskContext: (context: string) =>
```

Comment:
i dont think we're actually using this anywhere right?

Reply:
Just in the test right now.